styles/styles.css

```diff
@@ -15,35 +15,653 @@ li {
+  margin-bottom: 25px;
+}
+
+#hero-text {
+  text-align: left;
+  color: white;
+}
+
+#hero-text div {
+  position: absolute;
+  top: 25%;
+}
+/* ========================== End Hero Block ========================== */
+
+/* ========================== Tenant Block ========================== */
+#tenant-img {
+  grid-column: 1;
+  grid-row: 1;
+}
+
+#tenant-img > img {
+  width: 100%;
+}
+
+#tenant-text {
+  grid-column: 1;
+  grid-row: 2;
+}
+/* ========================== End Tenant Block ========================== */
+
+/* ========================== Advertising Block ========================== */
+#advertising-img {
+  grid-column: 1;
+  grid-row: 1;
+}
+
+#advertising-img > img {
+  width: 100%;
+}
+
+#advertising-text {
+  grid-column: 1;
+  grid-row: 2;
+}
+/* ========================== End Advertising Block ========================== */
+
+/* ========================== Agents Block ========================== */
+.agents-table {
+  width: 100%;
+  text-align: left;
+  border-collapse: collapse;
+  box-shadow: 0 0 0 1px var(--gray-color);
+  border-radius: 4px;
+  overflow: hidden;
+}
+
+.agents-table thead,
+tfoot {
+  background: var(--table-accent-color);
+}
+
+.agents-table th {
+  font-size: 1em;
+  font-weight: 700;
+  color: var(--primary-font-color);
+}
+
+.agents-table td {
+  font-size: 1em;
+  font-weight: 400;
+  color: var(--dark-color);
+}
+
+.agents-table tbody td {
+  border-top: 1px solid var(--gray-color);
+  border-bottom: 1px solid var(--gray-color);
+}
+
+.agents-table td,
+.agents-table th {
+  padding: 15px 20px;
+}
+
+.agents-table tfoot td i {
+  margin: 0 10px;
+  vertical-align: middle;
+}
+
+.agents-table tfoot td i:first-of-type {
+  margin-left: 30px;
+  color: var(--gray-color);
+}
+
+.agents-table caption {
+  margin: 5px 0;
+}
+/* ========================== End Agents Block ========================== */
+
+/* ========================== Footer ========================== */
+.footer-td div {
+  margin: 0 auto;
+  text-align: center;
+}
+
+footer {
+  background-color: var(--primary-font-color);
+  width: 100%;
+  padding: 20px 0;
+  text-align: center;
+}
+
+.copyright {
+  color: var(--primary);
+  font-size: 1em;
+  font-weight: 400;
+}
+
+.copyright a {
+  color: var(--primary);
+  text-decoration: underline;
+}
+/* ========================== End Footer ========================== */
+
+@media all and (max-width: 1100px) {
+  #logo {
+    width: 100%;
+    justify-content: space-between;
+  }
+
+  nav {
+    margin: 0;
+  }
+
+  .toggle + a,
+  .menu {
+    display: none;
+  }
+
+  .toggle {
+    display: block;
+    background-color: var(--primary);
+    padding: 14px 20px 14px 0;
+    color: var(--primary-font-color);
+    text-decoration: none;
+    border: none;
+    font-size: 0.9em;
+    text-transform: uppercase;
+    text-align: left;
+    font-weight: 600;
+  }
+
+  .toggle:hover {
+    background-color: var(--gray-color);
+  }
+
+  nav a {
+    padding-left: 0;
+  }
+
+  [id^="drop"]:checked + ul {
```

Comment:
A little bit of an overengineering using this complex identifier. Class works fine here (e.g. .menu-trigger:checked).